pkg/testutil/testutil.go

```diff
@@ -177,11 +178,11 @@ func assertGolden(t *testing.T, assertionType GoldenAssertion, path string, expe
 
 	switch assertionType {
 	case Text:
-		require.Equal(t, string(expected), string(actual), msg, path)
+		assert.Equal(t, string(expected), string(actual), msg, path)
 	case Bytes:
-		require.Equal(t, expected, actual, msg, path)
+		assert.Equal(t, expected, actual, msg, path)
 	case JSON:
-		require.JSONEq(t, string(expected), string(actual), msg, path)
+		assert.Equal(t, string(expected), string(actual), msg, path)
```

Comment:
This should be assert.JSONEq no? I'm amazed that tests are passing without that. I guess JSON marshaling is deterministic, so it's not that crazy. Error messages will be much worse without it though.